api/src/main/java/jakarta/json/JsonReader.java

```diff
@@ -52,6 +52,12 @@
  */
 public interface JsonReader extends  /*Auto*/Closeable {
 
+    /**
+     * Configuration property to forbid duplicate keys. The value of the property could be
+     * be anything.
+     */
+    public static final String FORBID_DUPLICATE_KEYS = "jakarta.json.JsonReader.forbidDuplicateKeys";
```

Comment:
Not here, this is jakarta API. I ment `org.glassfish.json.api` package - implementation specific API

Reply:
ah, okay. I'll make that change.

Reply:
@lukasj how would you like me to expose it from that package? In some sort of ConfigProperties interface?

Reply:
I created a new interface named `JsonConfig` in the `org.glassfish.json.api` package and moved the `FORBID_DUPLICATE_KEYS` constant there. Please see my latest commit.